doc/api/cli.md

```diff
@@ -565,6 +565,41 @@ const vm = require('node:vm');
 vm.measureMemory();
 ```
 
+### `--disable-wasm-trap-handler`
```

Comment:
I think it should be included in doc/node.1 too.

It's also missing:

*  the `added: REPLACEME` tag. 
*  Stability (optional)